internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -381,6 +381,52 @@ func isUserManagedNetwork(clusterInstall *hiveext.AgentClusterInstall) bool {
 	return false
 }
 
+func hyperthreadingInSpec(clusterInstall *hiveext.AgentClusterInstall) bool {
+	return len(clusterInstall.Spec.Compute) > 0 || clusterInstall.Spec.ControlPlane != nil
```

Comment:
how does compute > 0 and controlplane != nil imply hyperthreading? Could you add some documentation to this function?